source/particle/world.cc

```diff
@@ -1177,73 +1177,84 @@ namespace aspect
-                           "Upper limit for particle number per cell. This limit is "
-                           "useful for adaptive meshes to prevent coarse cells from slowing down "
-                           "the whole model. It will be checked and enforced after mesh "
-                           "refinement, after MPI transfer of particles and after particle "
-                           "movement. If there are "
-                           "\\texttt{n\\_number\\_of\\_particles} $>$ \\texttt{max\\_particles\\_per\\_cell} "
-                           "particles in one cell then "
-                           "\\texttt{n\\_number\\_of\\_particles} - \\texttt{max\\_particles\\_per\\_cell} "
-                           "particles in this cell are randomly chosen and destroyed.");
-        prm.declare_entry ("Particle weight", "10",
-                           Patterns::Integer (0),
-                           "Weight that is associated with the computational load of "
-                           "a single particle. The sum of particle weights will be added "
-                           "to the sum of cell weights to determine the partitioning of "
-                           "the mesh if the `repartition' particle load balancing strategy "
-                           "is selected. The optimal weight depends on the used "
-                           "integrator and particle properties. In general for a more "
-                           "expensive integrator and more expensive properties a larger "
-                           "particle weight is recommended. Before adding the weights "
-                           "of particles, each cell already carries a weight of 1000 to "
-                           "account for the cost of field-based computations.");
-        prm.declare_entry ("Update ghost particles", "false",
-                           Patterns::Bool (),
-                           "Some particle interpolation algorithms require knowledge "
-                           "about particles in neighboring cells. To allow this, "
-                           "particles in ghost cells need to be exchanged between the "
-                           "processes neighboring this cell. This parameter determines "
-                           "whether this transport is happening.");
-
-        Generator::declare_parameters<dim>(prm);
-        Integrator::declare_parameters<dim>(prm);
-        Interpolator::declare_parameters<dim>(prm);
-
-        Property::Manager<dim>::declare_parameters(prm);
-      }
-      prm.leave_subsection ();
+      constexpr unsigned int number_of_particle_worlds = 1;
+      for (unsigned int world_index = 0; world_index < number_of_particle_worlds; ++world_index)
+        {
+          if (world_index == 0)
+            {
+              prm.enter_subsection("Particles");
+            }
+          else
+            {
+              prm.enter_subsection("Particles " + std::to_string(world_index));
```

Comment:
I think it would be best if the subsections were ordered: "Particles", "Particles 2", ...

```suggestion
              prm.enter_subsection("Particles " + std::to_string(world_index+1));
```